subsys/mgmt/Kconfig

```diff
@@ -0,0 +1,96 @@
+# Kconfig - mcumgr configuration options
+
+#
+# Copyright Runtime.io 2018. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menu "Management"
+
+config MCUMGR_SMP_BT
+	bool
+	prompt "Bluetooth mcumgr SMP transport"
+	select MCUMGR
+	select BT
+	select BT_PERIPHERAL
+	default n
+	help
+	  Enables handling of SMP commands received over Bluetooth.
+
+config MCUMGR_SMP_SHELL
+	bool
+	prompt "Shell mcumgr SMP transport"
+	select MCUMGR
+	select UART_CONSOLE_MCUMGR
+	select CONSOLE_SHELL
+	select MBEDTLS
+	default n
+	help
+	  Enables handling of SMP commands received over shell.  This allows
+	  the shell to be use for both mcumgr commands and shell commands.
+
+if MCUMGR_SMP_SHELL
+config MCUMGR_SMP_SHELL_MTU
+	int
+	prompt "Shell SMP MTU"
+	default 256
+	help
+	  Maximum size of SMP frames sent and received over shell.  This value
+	  must satisfy the following relation:
+		MCUMGR_SMP_SHELL_MTU <= MCUMGR_BUF_SIZE + 2
```

Comment:
We have a build process that takes the help text and auto-generates configuration documentation by creating a .rst file.  This means the help text in the Kconfig needs to be ReST formatted, and as such, there needs to be a blank line before this indented line.